Index: db/ptss (1).sql
===================================================================
--- db/ptss (1).sql	(revision 1398542659a3aff02618cd7263f912c3d7464bef)
+++ db/ptss (1).sql	(revision 1398542659a3aff02618cd7263f912c3d7464bef)
@@ -1,243 +0,0 @@
--- phpMyAdmin SQL Dump
--- version 5.0.2
--- https://www.phpmyadmin.net/
---
--- Host: 127.0.0.1
--- Generation Time: Oct 20, 2020 at 03:25 PM
--- Server version: 10.4.13-MariaDB
--- PHP Version: 7.4.8
-
-SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
-START TRANSACTION;
-SET time_zone = "+00:00";
-
-
-/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
-/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
-/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
-/*!40101 SET NAMES utf8mb4 */;
-
---
--- Database: `ptss`
---
-
--- --------------------------------------------------------
-
---
--- Table structure for table `admin`
---
-
-CREATE TABLE `admin` (
-  `uname` varchar(10) NOT NULL,
-  `pass` varchar(10) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-
---
--- Dumping data for table `admin`
---
-
-INSERT INTO `admin` (`uname`, `pass`) VALUES
-('admin', 'admin');
-
--- --------------------------------------------------------
-
---
--- Table structure for table `chat_message`
---
-
-CREATE TABLE `chat_message` (
-  `chat_message_id` int(11) NOT NULL,
-  `to_user_id` int(11) NOT NULL,
-  `from_user_id` int(11) NOT NULL,
-  `chat_message` text NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-
--- --------------------------------------------------------
-
---
--- Table structure for table `product_image`
---
-
-CREATE TABLE `product_image` (
-  `product_id` int(11) NOT NULL,
-  `product_name` varchar(100) NOT NULL,
-  `product_category` varchar(50) NOT NULL,
-  `available_quantity` int(3) NOT NULL,
-  `product_description` varchar(500) NOT NULL,
-  `product_price` varchar(12) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-
---
--- Dumping data for table `product_image`
---
-
-INSERT INTO `product_image` (`product_id`, `product_name`, `product_category`, `available_quantity`, `product_description`, `product_price`) VALUES
-(4, 'beannie', 'Accessories & Shoes', 1, 'ring cap', 'RM 10.00'),
-(11, 'choc', 'Foods & Beverages', 3, 'chocojar', 'RM 10.00'),
-(12, 'case', 'Accessories & Shoes', 4, 'bestfriend casing', 'RM 8.50'),
-(14, 'makeup', 'Health & Beauty', 4, 'Glow makeup', '15.90');
-
--- --------------------------------------------------------
-
---
--- Table structure for table `profile`
---
-
-CREATE TABLE `profile` (
-  `profile_id` int(11) NOT NULL,
-  `profile_image` varchar(100) NOT NULL,
-  `email` varchar(100) NOT NULL,
-  `phone` int(12) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-
---
--- Dumping data for table `profile`
---
-
-INSERT INTO `profile` (`profile_id`, `profile_image`, `email`, `phone`) VALUES
-(1, '', 'ella7@gmail.com', 123456789);
-
--- --------------------------------------------------------
-
---
--- Table structure for table `register`
---
-
-CREATE TABLE `register` (
-  `id` int(11) NOT NULL,
-  `uname` varchar(12) NOT NULL,
-  `pass` varchar(12) NOT NULL,
-  `conpass` varchar(12) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-
--- --------------------------------------------------------
-
---
--- Table structure for table `staff`
---
-
-CREATE TABLE `staff` (
-  `id` int(11) NOT NULL,
-  `uname` varchar(20) NOT NULL,
-  `name` varchar(200) NOT NULL,
-  `pass` varchar(16) NOT NULL,
-  `conpass` varchar(12) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-
---
--- Dumping data for table `staff`
---
-
-INSERT INTO `staff` (`id`, `uname`, `name`, `pass`, `conpass`) VALUES
-(1, '000515100260', 'yasir', 'yas00', ''),
-(9, '000921070282', 'Arifah', '000921070282', ''),
-(11, '000227025402', 'Aina', '000227025402', '');
-
--- --------------------------------------------------------
-
---
--- Table structure for table `student`
---
-
-CREATE TABLE `student` (
-  `id` int(11) NOT NULL,
-  `uname` varchar(20) NOT NULL,
-  `name` varchar(200) NOT NULL,
-  `pass` varchar(10) NOT NULL,
-  `conpass` varchar(12) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
-
---
--- Dumping data for table `student`
---
-
-INSERT INTO `student` (`id`, `uname`, `name`, `pass`, `conpass`) VALUES
-(1, 'wana', '', 'wana00', ''),
-(2, '123', '123', '123', '123');
-
---
--- Indexes for dumped tables
---
-
---
--- Indexes for table `chat_message`
---
-ALTER TABLE `chat_message`
-  ADD PRIMARY KEY (`chat_message_id`);
-
---
--- Indexes for table `product_image`
---
-ALTER TABLE `product_image`
-  ADD PRIMARY KEY (`product_id`),
-  ADD UNIQUE KEY `product_name` (`product_name`);
-
---
--- Indexes for table `profile`
---
-ALTER TABLE `profile`
-  ADD PRIMARY KEY (`profile_id`),
-  ADD UNIQUE KEY `profile_image` (`profile_image`);
-
---
--- Indexes for table `register`
---
-ALTER TABLE `register`
-  ADD PRIMARY KEY (`id`);
-
---
--- Indexes for table `staff`
---
-ALTER TABLE `staff`
-  ADD PRIMARY KEY (`id`);
-
---
--- Indexes for table `student`
---
-ALTER TABLE `student`
-  ADD PRIMARY KEY (`id`);
-
---
--- AUTO_INCREMENT for dumped tables
---
-
---
--- AUTO_INCREMENT for table `chat_message`
---
-ALTER TABLE `chat_message`
-  MODIFY `chat_message_id` int(11) NOT NULL AUTO_INCREMENT;
-
---
--- AUTO_INCREMENT for table `product_image`
---
-ALTER TABLE `product_image`
-  MODIFY `product_id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=16;
-
---
--- AUTO_INCREMENT for table `profile`
---
-ALTER TABLE `profile`
-  MODIFY `profile_id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=2;
-
---
--- AUTO_INCREMENT for table `register`
---
-ALTER TABLE `register`
-  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT;
-
---
--- AUTO_INCREMENT for table `staff`
---
-ALTER TABLE `staff`
-  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=12;
-
---
--- AUTO_INCREMENT for table `student`
---
-ALTER TABLE `student`
-  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;
-COMMIT;
-
-/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
-/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
-/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/vcs.xml	(date 1603286635022)
+++ .idea/vcs.xml	(date 1603286635022)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: db/ptss 2020-10-20.sql
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- db/ptss 2020-10-20.sql	(date 1603202006944)
+++ db/ptss 2020-10-20.sql	(date 1603202006944)
@@ -0,0 +1,243 @@
+-- phpMyAdmin SQL Dump
+-- version 5.0.2
+-- https://www.phpmyadmin.net/
+--
+-- Host: 127.0.0.1
+-- Generation Time: Oct 20, 2020 at 03:25 PM
+-- Server version: 10.4.13-MariaDB
+-- PHP Version: 7.4.8
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `ptss`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `admin`
+--
+
+CREATE TABLE `admin` (
+  `uname` varchar(10) NOT NULL,
+  `pass` varchar(10) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `admin`
+--
+
+INSERT INTO `admin` (`uname`, `pass`) VALUES
+('admin', 'admin');
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `chat_message`
+--
+
+CREATE TABLE `chat_message` (
+  `chat_message_id` int(11) NOT NULL,
+  `to_user_id` int(11) NOT NULL,
+  `from_user_id` int(11) NOT NULL,
+  `chat_message` text NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `product_image`
+--
+
+CREATE TABLE `product_image` (
+  `product_id` int(11) NOT NULL,
+  `product_name` varchar(100) NOT NULL,
+  `product_category` varchar(50) NOT NULL,
+  `available_quantity` int(3) NOT NULL,
+  `product_description` varchar(500) NOT NULL,
+  `product_price` varchar(12) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `product_image`
+--
+
+INSERT INTO `product_image` (`product_id`, `product_name`, `product_category`, `available_quantity`, `product_description`, `product_price`) VALUES
+(4, 'beannie', 'Accessories & Shoes', 1, 'ring cap', 'RM 10.00'),
+(11, 'choc', 'Foods & Beverages', 3, 'chocojar', 'RM 10.00'),
+(12, 'case', 'Accessories & Shoes', 4, 'bestfriend casing', 'RM 8.50'),
+(14, 'makeup', 'Health & Beauty', 4, 'Glow makeup', '15.90');
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `profile`
+--
+
+CREATE TABLE `profile` (
+  `profile_id` int(11) NOT NULL,
+  `profile_image` varchar(100) NOT NULL,
+  `email` varchar(100) NOT NULL,
+  `phone` int(12) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `profile`
+--
+
+INSERT INTO `profile` (`profile_id`, `profile_image`, `email`, `phone`) VALUES
+(1, '', 'ella7@gmail.com', 123456789);
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `register`
+--
+
+CREATE TABLE `register` (
+  `id` int(11) NOT NULL,
+  `uname` varchar(12) NOT NULL,
+  `pass` varchar(12) NOT NULL,
+  `conpass` varchar(12) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `staff`
+--
+
+CREATE TABLE `staff` (
+  `id` int(11) NOT NULL,
+  `uname` varchar(20) NOT NULL,
+  `name` varchar(200) NOT NULL,
+  `pass` varchar(16) NOT NULL,
+  `conpass` varchar(12) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `staff`
+--
+
+INSERT INTO `staff` (`id`, `uname`, `name`, `pass`, `conpass`) VALUES
+(1, '000515100260', 'yasir', 'yas00', ''),
+(9, '000921070282', 'Arifah', '000921070282', ''),
+(11, '000227025402', 'Aina', '000227025402', '');
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `student`
+--
+
+CREATE TABLE `student` (
+  `id` int(11) NOT NULL,
+  `uname` varchar(20) NOT NULL,
+  `name` varchar(200) NOT NULL,
+  `pass` varchar(10) NOT NULL,
+  `conpass` varchar(12) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `student`
+--
+
+INSERT INTO `student` (`id`, `uname`, `name`, `pass`, `conpass`) VALUES
+(1, 'wana', '', 'wana00', ''),
+(2, '123', '123', '123', '123');
+
+--
+-- Indexes for dumped tables
+--
+
+--
+-- Indexes for table `chat_message`
+--
+ALTER TABLE `chat_message`
+  ADD PRIMARY KEY (`chat_message_id`);
+
+--
+-- Indexes for table `product_image`
+--
+ALTER TABLE `product_image`
+  ADD PRIMARY KEY (`product_id`),
+  ADD UNIQUE KEY `product_name` (`product_name`);
+
+--
+-- Indexes for table `profile`
+--
+ALTER TABLE `profile`
+  ADD PRIMARY KEY (`profile_id`),
+  ADD UNIQUE KEY `profile_image` (`profile_image`);
+
+--
+-- Indexes for table `register`
+--
+ALTER TABLE `register`
+  ADD PRIMARY KEY (`id`);
+
+--
+-- Indexes for table `staff`
+--
+ALTER TABLE `staff`
+  ADD PRIMARY KEY (`id`);
+
+--
+-- Indexes for table `student`
+--
+ALTER TABLE `student`
+  ADD PRIMARY KEY (`id`);
+
+--
+-- AUTO_INCREMENT for dumped tables
+--
+
+--
+-- AUTO_INCREMENT for table `chat_message`
+--
+ALTER TABLE `chat_message`
+  MODIFY `chat_message_id` int(11) NOT NULL AUTO_INCREMENT;
+
+--
+-- AUTO_INCREMENT for table `product_image`
+--
+ALTER TABLE `product_image`
+  MODIFY `product_id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=16;
+
+--
+-- AUTO_INCREMENT for table `profile`
+--
+ALTER TABLE `profile`
+  MODIFY `profile_id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=2;
+
+--
+-- AUTO_INCREMENT for table `register`
+--
+ALTER TABLE `register`
+  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT;
+
+--
+-- AUTO_INCREMENT for table `staff`
+--
+ALTER TABLE `staff`
+  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=12;
+
+--
+-- AUTO_INCREMENT for table `student`
+--
+ALTER TABLE `student`
+  MODIFY `id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=3;
+COMMIT;
+
+/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
+/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
+/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
Index: .idea/dataSources.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/dataSources.xml	(date 1603287192343)
+++ .idea/dataSources.xml	(date 1603287192343)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
+    <data-source source="LOCAL" name="contoh_ptss@localhost" uuid="035241dc-826a-495a-a626-b3ad1c0939b0">
+      <driver-ref>mariadb</driver-ref>
+      <synchronize>true</synchronize>
+      <jdbc-driver>org.mariadb.jdbc.Driver</jdbc-driver>
+      <jdbc-url>jdbc:mariadb://localhost:3306/contoh_ptss</jdbc-url>
+    </data-source>
+  </component>
+</project>
\ No newline at end of file
Index: database.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n$servername = \"localhost\";\r\n$username = \"root\";\r\n$password = \"\";\r\n$database = \"ptss\";\r\n\r\n// Create connection\r\n$conn = new mysqli($servername, $username, $password, $database);\r\nsession_start();
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- database.php	(revision 1398542659a3aff02618cd7263f912c3d7464bef)
+++ database.php	(date 1603286633742)
@@ -1,9 +1,9 @@
 <?php
 $servername = "localhost";
 $username = "root";
-$password = "";
-$database = "ptss";
+$password = "123456";
+$database = "contoh_ptss";
 
 // Create connection
 $conn = new mysqli($servername, $username, $password, $database);
-session_start();
\ No newline at end of file
+session_start();
Index: .idea/ptssmp.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/ptssmp.iml	(date 1603286634739)
+++ .idea/ptssmp.iml	(date 1603286634739)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/modules.xml	(date 1603286634815)
+++ .idea/modules.xml	(date 1603286634815)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/ptssmp.iml" filepath="$PROJECT_DIR$/.idea/ptssmp.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/codeStyles/codeStyleConfig.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/codeStyles/codeStyleConfig.xml	(date 1603286634991)
+++ .idea/codeStyles/codeStyleConfig.xml	(date 1603286634991)
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </state>
+</component>
\ No newline at end of file
diff --git db/readme.txt.txt db/readme.txt.txt
deleted file mode 100644
